thorlcr/activities/indexwrite/thindexwrite.cpp

```diff
@@ -326,6 +324,7 @@ class IndexWriteActivityMaster : public CMasterActivity
             mb.read(r);
             mb.read(slaveDuplicateKeyCount);
 
+            statsCollection.setStatistic(slaveIdx, StNumDuplicateKeys, slaveDuplicateKeyCount);
```

Comment:
I don't think this line is necessary.  I believe statsCollection should already have StNumDuplicateKeys.  On the slaves, mb.read(slaveDuplicateKeyCount) is serialized from CRuntimeStatisticCollection inactiveStats/activeStats.